userspace/libsinsp/filter.cpp

```diff
@@ -69,13 +69,13 @@ sinsp_filter_check_list::sinsp_filter_check_list()
 	//////////////////////////////////////////////////////////////////////////////
 	// ADD NEW FILTER CHECK CLASSES HERE
 	//////////////////////////////////////////////////////////////////////////////
-	add_filter_check(new sinsp_filter_check_fd());
-	add_filter_check(new sinsp_filter_check_thread());
 	add_filter_check(new sinsp_filter_check_event());
+	add_filter_check(new sinsp_filter_check_thread());
 	add_filter_check(new sinsp_filter_check_user());
 	add_filter_check(new sinsp_filter_check_group());
-	add_filter_check(new sinsp_filter_check_syslog());
 	add_filter_check(new sinsp_filter_check_container());
+	add_filter_check(new sinsp_filter_check_fd());
+	add_filter_check(new sinsp_filter_check_syslog());
```

Comment:
Note for reviewer: changing the order here makes the documentation more meaningful as the first field class shown is `event` (the common and most important one) instead of `fd`, but I don't know if this has any impact (incl. performance) on the rest of the engine